README.md

```diff
@@ -77,7 +82,23 @@ Belle is friends with Mrs. Potts
 Write your code here...
 
 ```ruby
-# code here
+friends = [
+  {
+    name: "Chip Potts",
+  },
+  {
+    name: "Cogsworth",
+  },
+  {
+    name: "Lumière",
+  {
+    name: "Mrs. Potts"
+  }]
+
+while
+  friends.each do |friend|
+    puts "Belle is friends with " + friends[:name] + "?"
+end
 ```
 
```

Comment:
Here's a solution:

```rb
friends.each do |friend|
  puts "Belle is friends with #{friend}"
end
```

So the intent of the exercise was to iterate through the provided array. You don't need a `while` there.